default/stringtables/en.txt

```diff
@@ -13410,11 +13425,17 @@ Additional experience and facilities improvements further extend the size of ind
 ##
 
 BLD_EVACUATION
-Evacuation System
+Evacuation
```

Comment:
Could be "Evacuate Planet" in consistent style with suggested "Abdandon Outpost"